ansible/roles/netbox/templates/usr/local/lib/netbox/configuration.py.j2

```diff
@@ -167,6 +175,13 @@ NAPALM_ARGS = {{ netbox__config_napalm_args | to_nice_json }}
 # Determine how many objects to display per page within a list. (Default: 50)
 PAGINATE_COUNT = {{ netbox__config_paginate_count }}
 
+# Enable installed plugins. Add the name of each plugin to the list.
+PLUGINS = {{ netbox__config_plugins | to_nice_json }}
+
+# Plugins configuration settings. These settings are used by various plugins that the user may have installed.
+# Each key in the dictionary is the name of an installed plugin and its value is a dictionary of settings.
+PLUGINS_CONFIG = {{ netbox__config_plugins_config | to_nice_json }}
+
```

Comment:
Thanks for updating the config. I think that in the future we should switch to the templating solution done in the `mailman` role, that way configuration options can be done conditionally, based on the NetBox version. But that can be dealt with in another pull request.

Reply:
Sounds like a plan. I agree.